charts/metering-ansible-operator/values.yaml

```diff
@@ -227,21 +227,25 @@ olm:
     minKubeVersion: "1.18.0"
     displayName: Metering
     description: |
-      The Metering Operator is a chargeback and reporting tool to provide accountability for how resources are used across a cluster. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and Cluster wide. The Metering Operator is a part of the [Kubernetes Reporting organization](https://coreos.com/blog/introducing-operator-framework-metering).
+      The Metering Operator is a generic reporting tool to provide accountability for how resources are used across a cluster cherry picking and storing information from different sources. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and Cluster for storage. The Metering Operator is a part of the [Kubernetes Reporting organization](https://github.com/kube-reporting).
 
       Read the user guide for more details on [running and viewing your first report](https://docs.openshift.com/container-platform/4.6/metering/metering-using-metering.html).
 
       ### Core capabilities
 
-      * **Chargeback/Showback** - Break down the reserved and utlized resources requested by applications.
+      * **Long term storage** - Break down the reserved and utlized resources requested by applications and store the information as long as needed.
 
       * **Pod, Namespace & Cluster Reports** - Built in reports exist to break down CPU and RAM in any way you desire.
-
-      * **Scheduled Reports** - Schedule reports to run on a standard interval, eg. daily or monthly
+ 
```

Comment:
This is rendering as a random space.

Reply:
I think there is some github glitch here.  If you view the file in github the changes are rendered across the several commits that touch L240, properly